src/main/java/cloud/orbit/spring/OrbitActorsProperties.java

```diff
@@ -117,4 +118,14 @@ public Long getDeactivationTimeoutInMilliseconds() {
     public void setDeactivationTimeoutInMilliseconds(final Long deactivationTimeoutInMilliseconds) {
         this.deactivationTimeoutInMilliseconds = deactivationTimeoutInMilliseconds;
     }
+
+    public Integer getExecutionPoolSize()
```

Comment:
We should look to integrate Lombok into project so we don't have to keep adding getter / setter methods.